instant-xml-macros/src/de.rs

```diff
@@ -4,9 +4,7 @@ use proc_macro2::{Ident, Literal, Span, TokenStream};
 use quote::quote;
 use syn::spanned::Spanned;
 
-use crate::meta::{ContainerMeta, FieldMeta, Mode, VariantMeta};
-
-use super::{discard_lifetimes, meta_items, Namespace};
+use crate::*;
```

Comment:
I don't like * imports, except for the single case of inline `tests` modules using `super::*`.

Reply:
Sorry, no. If you want me to review stuff, please optimize for reviewing as I suggest.